### ISSUE TITLE
Problem with prelude-evil: Symbol's function definition is void: global-evil-surround-mode

### ISSUE THREAD

**Issue by rubenmoor** [bbatsov/prelude#1031]
After a package update, my emacs was broken.

I have in the meantime installed emacs24 and prelude from scratch and just put this line into

$HOME/.emacs.d/personal/init.el

    (require 'prelude-evil)

Resulting in an error on start-up:

```
Warning (initialization): An error occurred while loading `/home/ruben/.emacs.d/init.elc':

Symbol's function definition is void: global-evil-surround-mode

To ensure normal operation, you should investigate and remove the
cause of the error in your initialization file.  Start Emacs with
the `--debug-init' option to view a complete error backtrace.
```

**Comment by rubenmoor** [comment]
Indeed a package update of evil-surround seems to have caused my issue.

I don't know how to downgrade, so as a workaround I removed `~/.emacs.d/evil-surround-20160907.1317` and replaced line 39  in `~/emacs.d/modules/prelude-evil.el`:

    (prelude-require-packages '(evil goto-chg evil-surround evil-visualstar evil-numbers))

with

    (prelude-require-packages '(evil goto-chg evil-visualstar evil-numbers))

**Comment by wbolster** [comment]
this issue is not prelude specific. it will disappear automatically once timcharper/evil-surround#83 is merged.

**Comment by wbolster** [comment]
the upstream issue has been fixed for a while; this issue is no longer relevant and can be closed

**Comment by bbatsov** [comment]
@wbolster Thanks!
